lib/services/dashd.js

```diff
@@ -5,7 +5,7 @@ var path = require('path');
 var spawn = require('child_process').spawn;
 var util = require('util');
 var dashcore = require('@dashevo/dashcore-lib');
-var zmq = require('zeromq');
+var zmq = require('@pshenmic/zeromq/v5-compat');
```

Comment:
Shouldn't we use same library as in platform?

Reply:
It is the same as in the platform's @dashevo/dapi package